controllers/operands/networkAddons_test.go

```diff
@@ -595,6 +595,26 @@ var _ = Describe("CNA Operand", func() {
 					expectedBaseDomain: "",
 				}),
 			)
+
+			It("Should deploy linux-bridge", func() {
+				existingCNAO, err := NewNetworkAddons(hco)
+				Expect(err).ToNot(HaveOccurred())
+
+				cl := commontestutils.InitClient([]client.Object{hco, existingCNAO})
+				handler := (*genericOperand)(newCnaHandler(cl, commontestutils.GetScheme()))
+				res := handler.ensure(req)
+				Expect(res.UpgradeDone).To(BeFalse())
+				Expect(res.Err).ToNot(HaveOccurred())
+
+				foundCNAO := &networkaddonsv1.NetworkAddonsConfig{}
+				Expect(
+					cl.Get(context.TODO(),
+						types.NamespacedName{Name: existingCNAO.Name, Namespace: existingCNAO.Namespace},
+						foundCNAO),
+				).To(Succeed())
+
+				Expect(foundCNAO.Spec.LinuxBridge).ToNot(BeNil(), "Linux Bridge spec should be added")
+			})
```

Comment:
Not sure this is not a bit overkill.

What about just call `NewNetworkAddons()`, and check if the `LinuxBridge` field is there?

Reply:
> Not sure this is not a bit overkill. I don't we need to initiate a reconcile loop here.
> 
> What do you think about just call `NewNetworkAddons()`, and check if the `LinuxBridge` field is there?

Indeed, done

> If we also want to check the new field in reconcile, we can remove the field before placing it in the fake client, then check that it restored by the handler. But I think this is a new test.

Imo the current is enough, wdyt?

Reply:
> Imo the current is enough, wdyt?

:+1: